packages/governance/src/committee.js

```diff
@@ -90,50 +114,94 @@ const start = (zcf, privateArgs) => {
     [...Array(committeeSize).keys()].map(makeCommitteeVoterInvitation),
   );
 
-  /** @type {AddQuestion} */
-  const addQuestion = async (voteCounter, questionSpec) => {
-    const quorumThreshold = quorumRule => {
-      switch (quorumRule) {
-        case QuorumRule.MAJORITY:
-          return ceilDivide(committeeSize, 2);
-        case QuorumRule.ALL:
-          return committeeSize;
-        case QuorumRule.NO_QUORUM:
-          return 0;
-        default:
-          throw Error(`${quorumRule} is not a recognized quorum rule`);
-      }
-    };
+  const makeCommitteeKit = defineHeapFarClassKit(
+    'Committee Facets',
+    CommitteeIKit,
+    initEmpty,
+    {
+      creatorFacet: {
+        getPoserInvitation() {
+          return getPoserInvitation(zcf, async (voteCounter, questionSpec) => {
+            const quorumThreshold = quorumRule => {
```

Comment:
AFAICT, the body of the argument function here is identical to the body of the `addQuestion` method below. Although the switch to inline concise methods admittedly make sharing this function harder, remember that you can send messages to self.

```suggestion
        getPoserInvitation() {
          const { facet: { creatorFacet } } = this;
          return getPoserInvitation(zcf, async (voteCounter, questionSpec) =>
            creatorFacet.addQuestion(voteCounter, questionSpec),
          );
        },
```

The suggestion above is if you stick with the `defineHeapFarClassKit`. If you switch to multiple `makeHeapFarInstance`s and call the lexical variable `creatorFacet`, then the thisful line disappears and the rest of the suggestion still applies.

Reply:
Thank you.  I should have noticed the duplication and looked for an appropriate way to get the sharing back.